NEWS.md

```diff
@@ -1,5 +1,16 @@
 ## 12.8.0 - Unreleased
 
+## 12.7.2 - Released (Poppy R2 2023)
+
+This release focused on fixing encumbrance status when changing fund distribution
```

Comment:
Please fix this comment accordingly based on latest ticket

Reply:
fixed